XenditExample/XenditExample/Controllers/ValidateCardViewController.swift

```diff
@@ -47,6 +47,11 @@ class ValidateCardViewController: UIViewController {
             return
         }
         
+        guard cardCvn != nil && cardNumber != nil && Xendit.isCvnValidForCardType(creditCardCVN: cardCvn!, cardNumber: cardNumber!) else {
```

Comment:
We should move to the validate method so customer doesn't have to do this themselves

Reply:
Ignore, we'd need to change the others too. Work for later.